common/src/main/java/com/sequenceiq/cloudbreak/logger/PropertyHelper.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.cloudbreak.logger;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class PropertyHelper {
+
+    private PropertyHelper() {
+    }
+
+    public static Boolean isJsonFormatEnabled() {
+        return Boolean.parseBoolean(getProperty("logger.format.json.enabled", null));
```

Comment:
Is there any reason why we are not using the @Value annotation of Spring like in all the other property holder components?

Reply:
that is for logging, it's needed right before Spring bootstrapping

Reply:
This is a spring boot application so even the logging framework is initialised by Spring boot framework. 
I would give it a try to use @Value annotation instead of this and let spring to build the dependencies in the right order(starting with logger dependencies)
On the other hand it would be a much cleaner code and you shouldn't have to deal with env variables and properties as @lacikaaa took this into the focus in his review.
what do you think @lacikaaa ?

Reply:
```

Since logging is initialized before the ApplicationContext is created, it is not possible to control logging from @PropertySources in Spring @Configuration files. The only way to change the logging system or disable it entirely is via System properties.
--
```
Sry, it is not supported, approving this PR